### PR TITLE
Register poynt adapter

### PR DESCRIPTION
**Pull request by icylisper**

### CODE REVIEW

File: test/pantheon/fixture_data.clj [hunk 1]
```diff
@@ -199,7 +199,8 @@
 (defn init-merchant [name]
   (assert (= :ok
              (-> (orpheus/request :merchant.create
-                                  {:name name})
+                                  {:name name
+                                   :config {:providers {:pos "poynt"}}})
```
Comment (kriyative) [inline test/pantheon/fixture_data.clj]: @icylisper - this doesn't really test the provider mechanism - how would we test it end-to-end? ... would it make sense to pull `pantheon` as a `test` profile dependency here and add HTTP level tests?

Comment (kriyative) [inline test/pantheon/fixture_data.clj]: I take the earlier comment back. That doesn't reflect the long term adapter development approach we were discussing earlier today.

I guess we really need to treat `pantheon-dev000` as a fixed test target for end-to-end tests in the adapters.

Comment (icylisper) [inline test/pantheon/fixture_data.clj]: Yes, it does not test the notification implementation (websockets etc). However, the test atleast resolves the provider cue and ignores if there are no ws connections.

Comment (icylisper) [inline test/pantheon/fixture_data.clj]: The actual end-to-end tests as per design will be in top-level Pantheon

**Pull request merged**